Content.Server/Weapon/Melee/EnergySword/Components/EnergySwordComponent.cs

```diff
@@ -0,0 +1,38 @@
+using Content.Shared.Sound;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.Maths;
+using System.Collections.Generic;
+
+namespace Content.Server.Weapon.Melee.Esword
+{
+    [RegisterComponent]
+    internal class EnergySwordComponent : Component
+    {
+        public override string Name => "Esword";
+
+        public Color BladeColor = Color.Blue;
```

Comment:
Default color is something not actually listed in the options?

Reply:
Sorry boss, you lost me here.
By that, do you mean that Color.Default should exist, or that I should manage to get the default color in the yaml file?

Reply:
The default is Color.Blue but it's not listed in colorOptions.